frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -1957,7 +1959,12 @@ private Map<String, Set<String>> getComponentFilterMap(PortletRequest request) {
                 }
                 Set<String> values = CommonUtils.splitToSet(parameter);
                 if (filteredField.equals(Component._Fields.NAME)) {
-                    values = values.stream().map(LuceneAwareDatabaseConnector::prepareWildcardQuery).collect(Collectors.toSet());
+                    if(!exactMatch.isEmpty() && !(parameter.startsWith("\"") && parameter.endsWith("\""))) {
```

Comment:
please maintain the correct formatting / intonation.
`if (` on line `1962` and `s -> "\"" + s + "\""` on line `1963`.

Kindly do the same change in `ProjectPortlet` and `ModerationPortlet` as well.